lib/rouge/lexers/php.rb

```diff
@@ -137,12 +143,17 @@ def self.detect?(text)
         end
 
         rule %r/(true|false|null)\b/, Keyword::Constant
+        rule /(?:void|\??(?:int|float|bool|string|iterable))\b/i, Keyword::Type
+        rule /\??(?:self|callable)\b/i, Keyword::Type
```

Comment:
Is there a reason to have this be in a separate rule?

Reply:
You're right, I'll fix/regroup that (originally I splitted these two to comment out the change since the first rule is PHP 7.0 and PHP 7.1 for the second)